src/tests/cli_tests.py

```diff
@@ -144,18 +152,44 @@ def raise_err(msg, log = None):
         print log
     raise NameError(msg)
```

Comment:
Just a nitpick but `NameError` isn't really an appropriate error to raise here. Best practice would be to add a small class that derives from `Exception` and takes that optional log parameter (and override `__str__` to print that out if provided).

Reply:
Thanks, @dewyatt . It was just a quick pick, will fix it to work in a better way at some point.